packages/website/docs/Internal/Invariants.md

```diff
@@ -1,19 +1,23 @@
 ---
-title: Statistical properties of algebraic combinations of distributions for property testing.
+title: Invariants
```

Comment:
Why remove the descriptive title?
Maybe, at least, ``Invariants of probability distributions``?

Reply:
The previous title was slimmer, speaking about algebraic combinations. I needed a broader title when I added the `inv \circ cdf = id` invariant.

Reply:
But still, those are about probability distributions

Reply:
your suggestion is in the next push.